docs/conf.py

```diff
@@ -104,26 +104,29 @@
 html_static_path = ['_static']
 html_css_files = ["theme_overrides.css"]
 
-html_theme_options = dict(
-    # canonical_url="",
-    repository_url="https://github.com/NCAR/geocat-examples",
-    repository_branch="main",
-    path_to_docs="docs",
-    use_edit_page_button=True,
-    use_repository_button=True,
-    use_issues_button=True,
-    home_page_in_toc=False,
-    navbar_footer_text="",
-    extra_footer=
-    "<em>This material is based upon work supported by the NSF National Center for Atmospheric Research, a major facility sponsored by the U.S. National Science Foundation and managed by the University Corporation for Atmospheric Research. Any opinions, findings and conclusions or recommendations expressed in this material do not necessarily reflect the views of the U.S. National Science Foundation.</em>",
-)
-
-# Specify logos
 html_theme_options = {
+    "repository_url":
+        "https://github.com/NCAR/geocat-examples",
+    "repository_branch":
+        "main",
+    "path_to_docs":
+        "docs",
+    "use_edit_page_button":
+        True,
+    "use_repository_button":
+        True,
+    "use_issues_button":
+        True,
+    "home_page_in_toc":
+        False,
+    "navbar_footer_text":
+        "",
     "logo": {
         "image_light": '_static/images/NSF_NCAR_light.svg',
         "image_dark": '_static/images/NSF_NCAR_dark.svg',
-    }
+    },
+    "extra_footer":
```

Comment:
Your PR description sounds like it removes the extra footer but isn't this it? Looks mostly like the formatting of `html_theme_options` changed

Reply:
Yeah, that wasn't the most clear and I don't think you saw this on applications.

It's the other way around.  The extra footer config and other other items weren't being applied because they were overwritten by the second dictionary definition.  

I could also have just modified the dictionary below to address this, but it seem cleaner just to merge it all into one and match how the dictionaries are defined in the theme docs.  That also introduces more changes though so I think it's a little harder to understand what's going on from the diffs.

Reply:
Pre-commit also formatted this in a way I don't love, but that's probably for another day.